database/insertData/laos/02_template_orgRegistration.js

```diff
@@ -0,0 +1,1003 @@
+                  owner_phone: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.ownerPhone.text", ""]
+                  }
+                  owner_email: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.ownerEmail.text", ""]
+                  }
+                  owner_fax: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.ownerFax.text", ""]
+                  }
+                  professional_id: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.profId.text", ""]
+                  }
+                  professional_experience: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.expList", ""]
+                  }
+                }
+              }
+              {
+                actionCode: "joinUserOrg"
+                trigger: ON_REVIEW_SUBMIT
+                sequence: 102
+                # TO-DO -- update condition to just check Outcome
+                # (from applicationData)
+                condition: {
+                  operator: "="
+                  children: [
+                    {
+                      operator: "objectProperties"
+                      children: [
+                        "applicationData.outcome"
+                      ]
+                    }
+                    "APPROVED"
+                  ]
+                }
+                parameterQueries: {
+                  user_id: {
+                    operator: "objectProperties"
+                    children: ["applicationData.userId"]
+                  }
+                  organisation_id: {
+                    operator: "objectProperties"
+                    children: ["outputCumulative.organisation.id"]
+                  }
+                  user_role: "Owner"
+                }
+              }
+              {
+                actionCode: "grantPermissions"
+                trigger: ON_REVIEW_SUBMIT
+                sequence: 103
+                condition: {
+                  operator: "="
+                  children: [
+                    {
+                      operator: "objectProperties"
+                      children: [
+                        "applicationData.outcome"
+                      ]
+                    }
+                    "APPROVED"
+                  ]
+                }
+                parameterQueries: {
+                  username: {
+                    operator: "objectProperties"
+                    children: ["applicationData.username"]
+                  }
+                  orgName: {
+                    operator: "objectProperties"
+                    children: ["outputCumulative.organisation.name"]
+                  }
+                  permissionNames: ["reviewJoinOrg", "applyOrgLicense"]
```

Comment:
Similar to my previous question for "reviewJoinOrg": 
Is the company that you can review been restricted already?

Reply:
https://github.com/openmsupply/application-manager-server/pull/400#discussion_r648794767